launcher/ui/pages/instance/ModFolderPage.cpp

```diff
@@ -195,8 +205,14 @@ void ModFolderPage::updateMods()
 
     auto mods_list = m_model->selectedMods(selection);
     bool use_all = mods_list.empty();
-    if (use_all)
+    if (use_all) {
```

Comment:
because the for loop is inside this if statement, the disabling will not work with selected mods. I don't think this should be the case, as it's quite weird to have the option be ignored in some cases, especially considering the user will probably expect that checking for updates with no mods selected is equal to checking for updates with all mods selected on the ui.

Reply:
Yeah, I just felt that it would be good if this could be easily bypassed without changing the setting. That would be useful if you know the latest version works with the modpack but don't want it to update in the future.